Tests/ApolloCodegenTests/CodeGeneration/Templates/InputObjectTemplateTests.swift

```diff
@@ -2540,4 +2545,67 @@ class InputObjectTemplateTests: XCTestCase {
     }
   }
   
+  // MARK: - Schema Customization Tests
+  
+  func test__render__givenInputObjectAndField_withCustomNames_shouldRenderWithCustomNames() throws {
+    // given
+    let customInputField = GraphQLInputField.mock(
+      "myField",
+      type: .nonNull(.string()),
+      defaultValue: nil
+    )
+    customInputField.name.customName = "myCustomField"
+    buildSubject(
+      name: "MyInputObject",
+      customName: "MyCustomInputObject",
+      fields: [
+        GraphQLInputField.mock(
+          "fieldOne",
+          type: .nonNull(.string()),
+          defaultValue: nil
+        ),
+        customInputField
+      ]
+    )
+
+    let expected = """
+    \(subject.graphqlInputObject.name.typeNameDocumentation)
+    public struct MyCustomInputObject: InputObject {
+      public private(set) var __data: InputDict
+
+      public init(_ data: InputDict) {
+        __data = data
+      }
+
+      public init(
+        fieldOne: String,
+        myCustomField: String
+      ) {
+        __data = InputDict([
+          "fieldOne": fieldOne,
+          "myField": myCustomField
+        ])
+      }
+
+      public var fieldOne: String {
+        get { __data["fieldOne"] }
+        set { __data["fieldOne"] = newValue }
+      }
+
+      // This type has been renamed from the schema type 'myField'
+      // using schema customization configuration.
+      public var myCustomField: String {
+        get { __data["myField"] }
+        set { __data["myField"] = newValue }
+      }
```

Comment:
Same opinion here:

```suggestion
      // Renamed from GraphQL schema value: 'myField'      
      public var myCustomField: String {
        get { __data["myField"] }
        set { __data["myField"] = newValue }
      }
```

Reply:
You're also using the generic message "This _type_ has been renamed", but enum cases and fields on input objects aren't "types", which is why I just changed it to the generic term "value".